modules/proj4/src/lib/proj4-projection.d.ts

```diff
@@ -0,0 +1,24 @@
+import proj4 from 'proj4';
+
+/**
+ * Predefined Aliases
+ * 'WGS84':       'EPSG:4326'
+ * 'EPSG:3785':   'EPSG:3857' 'EPSG:4269'
+ * 'GOOGLE':      'EPSG:3857' 'EPSG:4269'
+ * 'EPSG:900913': 'EPSG:3857' 'EPSG:4269'
+ * 'EPSG:102113': 'EPSG:3857' 'EPSG:4269'
```

Comment:
Might be good to put these in the docs.

| predefined projections | aliases |
| --|--|
| `EPSG:4326` | `WGS84` |
| `EPSG:4269` | |
| `EPSG:3857` | `EPSG:3785`, `GOOGLE`, `EPSG:900913`, `EPSG:102113` |

Reply:
Yes I want to do another pass on the docs (would certainly welcome some help). One of the shortcomings of proj4js is its relatively sparse documentation.

Some links to WKT-CRS specs, a list of built-in projections etc would be nice.

Reply:
I think [this](http://docs.opengeospatial.org/is/18-010r7/18-010r7.html) is the official WKT-CRS specification, but it's huge and unreadable. I'd link to the [Wikipedia section](https://en.wikipedia.org/wiki/Well-known_text_representation_of_coordinate_reference_systems#Coordinate_reference_systems), and then also maybe list epsg.io and spatialreference.org as references.

E.g. if you go to https://epsg.io/4326, at the bottom of the page you can see its representation in WKT:
```
GEOGCS["WGS 84",
    DATUM["WGS_1984",
        SPHEROID["WGS 84",6378137,298.257223563,
            AUTHORITY["EPSG","7030"]],
        AUTHORITY["EPSG","6326"]],
    PRIMEM["Greenwich",0,
        AUTHORITY["EPSG","8901"]],
    UNIT["degree",0.0174532925199433,
        AUTHORITY["EPSG","9122"]],
    AUTHORITY["EPSG","4326"]]
```